index.html

```diff
@@ -2290,9 +2299,7 @@ <h2 id="docconformance">
               [^tr^]
             </td>
             <td>
-              <a href="#index-aria-row">`role=row`</a>, may be
-              explicitly declared when child of a `table` element
-              with `role=grid` or `treegrid`
+              <a href="#index-aria-row">`role=row`</a>
```

Comment:
this *should* be what we need here.  again, we want to just make sure that there aren't instances where this is not exposed correctly, so as to not remove a special case for why authors would need to assign the implicit role.